src/store/selectors/tokens.js

```diff
@@ -0,0 +1,31 @@
+import { createSelector } from 'reselect';
+
+const state = s => s.get('tokens');
+
+const defaultTokenPair = createSelector(
+  state,
+  s => s.get('defaultTokenPair')
```

Comment:
Ok .. again what is this S... I'm not familiar with this createSelector so seeing this S... rings no bell to me

Reply:
*s* would always stand for the state. i can use full string if You prefer.